cmdr/src/giti/branch/delete.rs

```diff
@@ -293,3 +295,65 @@ pub fn try_execute_git_command_to_get_branches() -> CommonResult<Vec<String>> {
         }
     }
 }
+
+// Get all the branches to check out to. prefix current branch with `(current)`.
+pub fn get_branches() -> CommonResult<Vec<String>> {
+    let branches = try_execute_git_command_to_get_branches()?;
+    // If branch name is current_branch, then append `(current)` in front of it.
+    // Create command.
+    let mut command = Command::new("git");
+    let command: &mut Command = command.args(["branch", "--show-current"]);
+
+    let result_output = command.output();
+
+    let current_branch;
+    match result_output {
+        // Can't even execute output(), something unknown has gone wrong. Propagate the
+        // error.
+        Err(error) => {
+            return report_unknown_error_and_propagate(command, error);
+        }
+        Ok(output) => {
+            let output_string = String::from_utf8_lossy(&output.stdout);
+            current_branch = output_string.to_string().trim_end_matches('\n').to_string();
+            log_debug(format!(
```

Comment:
- for errors, it is best to use log_error() and not log_debug()
- when logs are optional (eg: if they are log_debug() or log_info()), then best to wrap them in a call_if_true!... macro. the flag that is used in this macro eg: call_if_true!(DEBUG_TUI_MOD...) can be used to turn on / off all logging related to that "kind" of log message.